marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx

```diff
@@ -89,30 +89,47 @@ describe('Entity Modeling', () => {
 
     propertyTable.editProperty('fname');
     cy.waitUntil(() => propertyModal.getToggleStepsButton().should('exist')).click();
-    cy.contains('mapPersonJSON');
-    cy.contains('match-person');
-    cy.contains('merge-person');
-    cy.contains('master-person');
-    cy.contains('Hide Steps...');
+    cy.contains('mapPersonJSON').should('be.visible');
+    cy.contains('match-person').should('be.visible');
+    cy.contains('merge-person').should('be.visible');
+    cy.contains('master-person').should('be.visible');
+    cy.contains('Hide Steps...').should('be.visible');
+
+    cy.contains('Show Steps...').should('not.be.visible');
+
     propertyModal.getToggleStepsButton().click();
-    cy.contains('Show Steps...');
+
+    cy.contains('Show Steps...').should('be.visible');
+
+    cy.contains('mapPersonJSON').should('not.be.visible');
+    cy.contains('match-person').should('not.be.visible');
+    cy.contains('merge-person').should('not.be.visible');
+    cy.contains('master-person').should('not.be.visible');
+    cy.contains('Hide Steps...').should('not.be.visible');
     propertyModal.getCancelButton().click();
 
     entityTypeTable.getDeleteEntityIcon('Person').click();
-    cy.contains('Entity type is used in one or more steps.');
-    cy.contains('Show Steps...');
+    cy.contains('Entity type is used in one or more steps.').should('be.visible');
+    cy.contains('Show Steps...').should('be.visible');
+    cy.contains('Hide Steps...').should('not.be.visible');
 
     confirmationModal.getToggleStepsButton().click();
-    cy.contains('mapPersonJSON');
-    cy.contains('Hide Steps...');
+    cy.contains('mapPersonJSON').should('be.visible');
+    cy.contains('Hide Steps...').should('be.visible');
+    cy.contains('Show Steps...').should('not.be.visible');
 
     confirmationModal.getCloseButton(ConfirmationType.DeleteEntityStepWarn).click();
     entityTypeTable.getEntity('Person').should('exist');
   });
 
-  it('can create a new entity, add relationship type, and add identifier confirmation, delete property from modal, and delete entity', () => {
-    modelPage.getAddEntityButton().should('exist');
-    modelPage.getAddEntityButton().click();
+  it('can check for duplicate entity, create a new entity, add relationship type, and add identifier confirmation, delete property from modal, and delete entity', () => {
+    cy.waitUntil(() => modelPage.getAddEntityButton()).click();
+    entityTypeModal.newEntityName('Person');
+    entityTypeModal.getAddButton().click();
+    cy.contains('An entity type already exists with a name of Person').should('be.visible');
+    entityTypeModal.getAddButton().should('not.be.disabled');
```

Comment:
I would prefer to update existing unit test 'Creating duplicate entity shows error message' to verify Add button is enabled, than verifying this in e2e. This is ok now and something to keep in mind for future tests.

Reply:
I still think we need this test since the message depends on the server response.

Reply:
ok.